src/utils/filterPeople.ts

```diff
@@ -0,0 +1,65 @@
+import { Person } from '../types';
+
+export const filterPeople = (
+  people: Person[],
+  sex: string | null,
+  query: string | null,
+  centuries: string[],
+  sortField: string | null,
+  order: string | null,
+) => {
+  let tempPeople = [...people];
```

Comment:
The `filter` method returns a copy of the array, so you don't need to do it manually.